guides/v2.3/rest/tutorials/msi-order-processing/create-shipment.md

```diff
@@ -150,38 +104,10 @@ Use the same endpoint to ship the remaining 35 `sp2` items from the Reno warehou
 
 **Response**
 
-``` json
-{
-    "created_at": "2018-08-02 22:22:23",
-    "customer_id": 2,
-    "entity_id": 16,
-    "increment_id": "2000000016",
-    "order_id": 1,
-    "packages": [],
-    "shipment_status": 1,
-    "total_qty": 35,
-    "updated_at": "2018-08-02 22:22:23",
-    "items": [
-        {
-            "entity_id": 10,
-            "name": "Simple Product 2",
-            "parent_id": 16,
-            "price": 10,
-            "product_id": 8,
-            "sku": "sp2",
-            "order_item_id": 2,
-            "qty": 35
-        }
-    ],
-    "tracks": [],
-    "comments": [],
-    "extension_attributes": {
-        "source_code": "reno_wh"
-    }
-}
-```
+The shipment ID, such as `19`.
+
 
 ## Verify this step {#verify-step}
 
 1. Click **Sales** > **Shipments**. The two shipments for this order are displayed in the grid.
-2. Click **Catalog** > **Products** to verify **Quantity per Source** values for each product has been deducted according to the selections made at shipment. 
+2. Click **Catalog** > **Products** to verify **Quantity per Source** values for each product has been deducted according to the selections made at shipment.
```

Comment:
"has been deducted" is not only passive, but reads weird in this step. Must be missing something...

Reply:
Hmm could use something more like: Click **Catalog** > **Products** to verify **Quantity per Source** values for each product. The quantities per source display updated totals according to the selections made at shipment.